packages/fields/src/fields/author/index.tsx

```diff
@@ -15,6 +15,7 @@ const authorField: Field< BasePostWithEmbeddedAuthor > = {
 	id: 'author',
 	type: 'integer',
 	elements: [],
+	supportsBulk: true,
```

Comment:
Should a `type` also be able to decide if it supports bulk or not?
Maybe only if no custom `Edit` is being passed in?